SSA/Experimental/Bits/Fast/Tactic.lean

```diff
@@ -199,3 +255,6 @@ def test24 (x y : BitVec 2) : (x ||| y) = (( x &&& (~~~y)) + y) := by
 
 def test25 (x y : BitVec 2) : (x &&& y) = (((~~~x) ||| y) - ~~~x) := by
   bv_automata
+
+def test26 {w : Nat} (x y : BitVec (w + 1)) : 1 + x + 0 = 1  + x := by
+  bv_automata
```

Comment:
Maybe add a test case that has constants but is fixed-bitwidth and one that is arbitrary-bitwidth but does not use constants?